src/Target.php

```diff
@@ -259,23 +270,25 @@ public function getMessagePrefix(array $message): string
      * }
      * ```
      */
-    public function setEnabled($value): void
+    public function setEnabled($value): self
     {
-        $this->_enabled = $value;
+        $this->enabled = $value;
+
+        return $this;
     }
 
     /**
      * Check whether the log target is enabled.
-     * @property bool Indicates whether this log target is enabled. Defaults to true.
      * @return bool A value indicating whether this log target is enabled.
+     * @property bool Indicates whether this log target is enabled. Defaults to true.
      */
     public function getEnabled(): bool
```

Comment:
Can we rename it to `isEnabled()` with two setters: `enable()` and `disable()`?

Reply:
Done. I also kept the setEnabled() setter to be able to pass a callback